modules/cluster-logging-elasticsearch-limits.adoc

```diff
@@ -34,11 +37,10 @@ spec:
       elasticsearch:
         resources: <1>
           limits:
-            cpu: "4000m"
-            memory: "4Gi"
+            memory: "16Gi"
           requests:
-            cpu: "4000m"
-            memory: "1Gi"
+            cpu: "1"
```

Comment:
Why are we reducing the number of available CPUs for Elasticsearch to `1` by default?

Reply:
The default provided by the operator is 1 and has been since 3.9

Reply:
That seems pretty bad.  How is the default changed?